ova/Vagrantfile

```diff
@@ -10,5 +10,5 @@ Vagrant.configure("2") do |config|
     vb.memory = "4096"
     vb.cpus = "4"
   end
-  config.vm.provision :shell, path: "provision.sh", :args => "#{ENV['OVA_WAZUH_VERSION']} #{ENV['OVA_ELK_VERSION']}  #{ENV['STATUS_PACKAGES']} #{ENV['DIRECTORY']}"
+    config.vm.provision :shell, path: "provision.sh", :args => "#{ENV['WAZUH_VERSION']} #{ENV['OPENDISTRO_VERSION']}  #{ENV['ELK_VERSION']}  #{ENV['STATUS']} #{ENV['BRANCH']} -d"
```

Comment:
```suggestion
  config.vm.provision :shell, path: "provision.sh", :args => "#{ENV['WAZUH_VERSION']} #{ENV['OPENDISTRO_VERSION']}  #{ENV['ELK_VERSION']}  #{ENV['STATUS']} #{ENV['BRANCH']} -d"
```

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a